### PR TITLE
Minor changes to OpenELB

### PR DESCRIPTION
**Pull request by Bettygogo2021**
Hi, Feynman, pls kindly help me merge the updates. 3ks.
/assign @FeynmanZhou

### CODE REVIEW

File: content/en/blogs/Kubernetes-multicluster-KubeSphere.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+title: 'Manage Multiple Clusters and Deploy Applications for Kubernetes in Hybrid Cloud'
```
Comment (FeynmanZhou) [inline content/en/blogs/Kubernetes-multicluster-KubeSphere.md:2]: I prefer this title: Kubernetes Multi-cluster management and application deployment in Hybrid Cloud.

**Review by FeynmanZhou — COMMENTED** [review]
See my comment above, thanks.

**Comment by Bettygogo2021** [comment]
Pengfei, I've changed the title as suggested. Pls check it.
/assign @FeynmanZhou

**Pull request merged**